tests/farm/LiquidityMining.js

```diff
@@ -315,6 +316,14 @@ contract("LiquidityMining", (accounts) => {
             expect(_unlockedImmediatelyPercent).bignumber.equal(newUnlockedImmediatelyPercent);
         });
 
+        it("successfully set to 10000 (100%)", async () => {
```

Comment:
cannot find the test for setting by admin to 0

Reply:
updated